validator/node/node.go

```diff
@@ -41,9 +41,9 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
-// ValidatorClient defines an instance of an eth2 validator that manages
+// ValidatorNode defines an instance of an eth2 validator that manages
```

Comment:
`ValidatorClient` is popularly used across eth2 community since day 0. I'm against this change in code even it strives for some what better consistency. It's more important that we align with the common terminology

Reply:
Agree with keeping it as a validator client as it is a common name across all implementations